app/views/devise/sessions/new.html.slim

```diff
@@ -1,25 +1,28 @@
-h2
-  | Log in
-
-= render 'shared/notifications'
```

Comment:
created https://github.com/kubic-project/velum/issues/70

@kwwii i think we'd need some input from you there